examples/pma/imu.py

```diff
@@ -0,0 +1,14 @@
+from pitop.pma import Imu
```

Comment:
Would prefer to rename the examples to explain what they are - `imu.py` and `imu2.py` are not very descriptive